src/shell/Shell.cpp

```diff
@@ -923,6 +928,28 @@ PersistentRefHolder<ContextRef> createEscargotContext(VMInstanceRef* instance, b
     return context;
 }
 
+uint64_t g_lastTick = 0;
+
+static uint64_t fastTickCount()
+{
+#if defined(CLOCK_MONOTONIC_COARSE)
+    timespec ts;
+    clock_gettime(CLOCK_MONOTONIC_COARSE, &ts);
+    return (uint64_t)ts.tv_sec * 1000UL + ts.tv_nsec / 1000000UL;
+#else
+    return tickCount();
+#endif
+}
+
+static void PrintGCMemory(void* data)
+{
+    uint64_t currentTick = fastTickCount() ;
+    if (currentTick - g_lastTick >= 1000) {
+        g_lastTick = currentTick;
+        fprintf(static_cast<FILE*>(data), "Tick %" PRIu64 ": Memory usage[%f MB], Heap usage[%f MB]\n", fastTickCount(), GC_get_memory_use() / 1024.f / 1024.f, GC_get_heap_size() / 1024.f / 1024.f);
```

Comment:
Heap usage measurement is enough for now(`GC_get_memory_use()` could be removed).
And instead of directly invoking gc method, how about using `Memory::heapSize()` api?

Reply:
And one more thing,
Would you remove the following line too?
https://github.com/Samsung/escargot/blob/af792911596304c62193fe490f7711ae66d1c826/src/shell/Shell.cpp#L722
This print line is needless. Thanks:)

Reply:
Thank you, I applied this.